src/components/integrated/OakLessonNavItem/OakLessonNavItem.tsx

```diff
@@ -0,0 +1,304 @@
+  BaseOakLessonNavItemProps<C> & SectionProps;
+
+const StyledLabel = styled(OakBox)``;
+
+const StyledRoundIcon = styled(OakRoundIcon)<{
+  $isDisabled?: boolean;
+}>`
+  width: ${parseSpacing("all-spacing-8")};
+  height: ${parseSpacing("all-spacing-8")};
+  padding: 0;
+
+  background: transparent;
+
+  img {
+    filter: ${(props) =>
+      parseColorFilter(props.$isDisabled ? "icon-disabled" : "icon-inverted")};
+  }
+`;
+
+const StyledLessonNavItem = styled(OakFlex)<{ $isDisabled?: boolean }>`
+  outline: none;
+  text-align: initial;
+
+  &:focus-visible {
+    box-shadow: ${parseDropShadow("drop-shadow-centered-yellow")},
+      ${parseDropShadow("drop-shadow-centered-grey")};
+  }
+
+  ${(props) => props.$isDisabled && "cursor: default"}
+
+  ${(props) =>
+    !props.$isDisabled &&
+    css`
+      cursor: pointer;
+
+      &:hover,
+      &:active {
+        ${StyledLabel} {
+          text-decoration: underline;
+        }
+
+        ${StyledRoundIcon} {
+          background: ${parseColor("bg-btn-primary")};
+
+          img {
+            filter: ${parseColorFilter("icon-main")};
+          }
+        }
+      }
+
+      &:active {
+        box-shadow: ${parseDropShadow("drop-shadow-yellow")},
+          ${parseDropShadow("drop-shadow-grey")};
+      }
+    `}
+`;
+
+/**
+ * Enables navigation to the given section of a lesson as well as displaying current progress
+ */
+export const OakLessonNavItem = <C extends ElementType = "a">(
+  props: OakLessonNavItemProps<C>,
+) => {
+  const { as, lessonSectionName, progress, isDisabled, ...rest } = props;
+  const [notStartedBackgroundColor, backgroundColor, borderColor] =
+    pickColorsForSection(lessonSectionName);
+
+  return (
+    <StyledLessonNavItem
+      as={as ?? "a"}
+      $gap="space-between-m"
+      $alignItems="center"
+      $background={
+        progress === "not-started" ? notStartedBackgroundColor : backgroundColor
+      }
+      $ph={["inner-padding-m", "inner-padding-xl"]}
+      $pv="inner-padding-l"
+      $borderRadius="border-radius-l"
+      $borderColor={borderColor}
+      $ba="border-solid-l"
+      $isDisabled={isDisabled}
+      {...rest}
+    >
+      <OakFlex $width="all-spacing-13" $justifyContent="center">
+        <OakIcon
+          iconName={pickIconForSection(lessonSectionName)}
+          $width="all-spacing-10"
+          $height="all-spacing-10"
+        />
+      </OakFlex>
+      <OakBox style={{ flex: 1 }}>
+        <StyledLabel
+          as="strong"
+          $font={["heading-6", "heading-5"]}
+          $color={isDisabled ? "text-disabled" : "text-primary"}
+        >
+          {pickLabelForSection(lessonSectionName)}
+        </StyledLabel>
+        <OakBox $font={["body-2", "body-1"]}>
+          {pickSummaryForProgress(props)}
+        </OakBox>
+      </OakBox>
+      {renderQuestionCounter(props)}
+      <StyledRoundIcon iconName="chevron-right" $isDisabled={isDisabled} />
+    </StyledLessonNavItem>
```

Comment:
This could easily be extracted into a more general `OakNavItem` in a follow-up PR if this is something that makes sense as a reusable piece.

Reply:
That could be useful for the future . (see my previous comment)